src/pages/docs/test-management/atto-in-action/overview.md

```
@@ -0,0 +1,64 @@
+- **Generator Agent**: Creates test cases automatically based on application behavior and user workflows.
+
+- **Executor Agent**: Runs test cases in test runs with optimized speed and efficiency.
+
+- **Bug Reporter Agent**: Reports bugs to external tools with all relevant details.
+
+---
+
+## **Getting Started**
+
+### **Generator Agent**
+   - [Generate Test Cases with Prompts](https://testsigma.com/docs/test-management/atom-in-action/generator-agent/generate-tests-with-prompts/)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)